packages/x-charts-vendor/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-charts-vendor",
-  "version": "7.16.0",
+  "version": "7.17.0",
```

Comment:
@mui/x 

What do we do in cases where a package version is out of sync with others?

Others are being bumped to `7.18.0`, but this is still on `7.16.0`.

Reply:
It should be synced.
This means that on last release there was no need to release it, so we skipped it, but whenever we are releasing—we use the version of the release.